private-distribution.yml

```diff
@@ -185,7 +185,7 @@ Resources:
         #
         # So we should try switching to using their application in the
         # serverlessrepo instead of ours. 
-        ApplicationId: arn:aws:serverlessrepo:us-east-1:816253370536:applications/cloudfront-authorization-at-edge
+        ApplicationId: arn:aws:serverlessrepo:us-east-1:520945424137:applications/cloudfront-authorization-at-edge
```

Comment:
Can you update the comment above this line?
I'd suggest leaving a reference to the old app in the comment. Something like: 
> we initially used a customized version of the AWS app: 
> - "arn:aws...".  
> - "github repo url"
> 
> Our customizations were merged upstream so this is now using the official app.

Reply:
Ah, right. Thanks for catching that.

Does the latest look good?

Reply:
Yup looks good. 👍